packages/design-system-scripts/gulp/build.js

```diff
@@ -88,7 +88,7 @@ async function copyAll(dir, options) {
 async function generateTypeDefinitions(dir, changedPath) {
   const src = path.join(dir, 'src', 'components');
   const srcGlob = changedPath
-    ? [changedPath`!${src}/**/*.{js,jsx}`]
```

Comment:
I think what Bernard was trying to do here is that if changedPath is included, only use it if it's typescript file (not a js, jsx file). It didn't work though because this line threw an error when rebuilding

(node:97074) UnhandledPromiseRejectionWarning: TypeError: changedPath is not a function
    at generateTypeDefinitions (/home/kimieann/projects/design-system/packages/design-system-scripts/gulp/build.js:91:19)
    at /home/kimieann/projects/design-system/packages/design-system-scripts/gulp/build.js:187:16
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at /home/kimieann/projects/design-system/packages/design-system-scripts/gulp/watch.js:52:7
(node:97074) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)